db/clinic/routers/doctor.py

```
@@ -15,6 +15,10 @@
 def get_doctors(db: Session = Depends(get_db)):
     return doctor.get_all_doctors(db)
 
+@router.get("/specialization", response_model=List[schemas.DoctorShow])
+def get_doctors(specialization: str, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Redefinition of unused `get_doctors` from line 15

(F811)